index.html

```diff
@@ -0,0 +1,148 @@
+      <div id="mobMenuContainer">
+        <ul class="mobile-menu">
+          <button class="close-menu-btn"><i class="fa-solid fa-xmark closeup"></i></button>
+          <li><i class="fa-solid fa-school-circle-check"></i><a href="about.html">About</a></li>
+          <li><i class="fa-solid fa-book"></i><a href="#course-features">Methodology</a></li>
+          <li><i class="fa-solid fa-person-chalkboard"></i></i><a href="#feat-teachers">Teachers </a></li>
+          <li><i class="fa-solid fa-comments"></i></i><a href="about.html#testimonials">Testimonials</a></li>
+          <li><i class="fa-solid fa-id-card-clip"></i><a href="">Enrol Now </a></li>
+          <span class="br-med"></span>
+          <li><i class="fa-solid fa-table-list"></i><a href="">My Dashboard</a></li>
+          <li><i class="fa-solid fa-arrow-right-from-bracket"></i><a href="">Logout</a></li>
+          <span class="br-med"></span>
+          <li><i class="fa-brands fa-facebook-f"></i><a href="">Facebook</a></li>
+          <li><i class="fa-brands fa-instagram"></i><a href="">Instagram</a></li>
+          <a href="about.html#logo-space" id="logo-mob-menu">
+            <img src="assets/logo/white-small.png" alt="Eagle English Logo">
+          </a>
+        </ul>
+      </div>
+    </nav>
+    <!-- -------- -->
+    <!-- DESKTOP NAV BAR -->
+    <nav id="pre-nav">
+      <ul>
+        <li><a href="">
+            <i class="fa-brands fa-facebook-f"></i>
+          </a></li>
+        <li><a href="">
+            <i class="fa-brands fa-instagram"></i>
+          </a></li>
+        <li><a href="">English </a></li>
+        <li><a href="">My Dashboard</a></li>
+        <li><a href="">Logout</a></li>
+      </ul>
+    </nav>
+    <nav id="desktop-nav">
+      <a id="logo" href="">
+        <img src="assets/logo/dark-small.png" alt="Eagle English">
+      </a>
+      <ul>
+        <li><a href="about.html">About</a></li>
+        <li><a href="#course-features">Methodology</a></li>
+        <li><a href="#feat-teachers">Teachers </a></li>
+        <li><a href="about.html#testimonials">Testimonials</a></li>
+        <li><a href="#our-clients">Our Clients</a></li>
+        <li><button>Enrol Now</button></li>
+      </ul>
+    </nav>
+  </header>
+
+  <main>
+    <!-- INTRO -->
+    <section id="intro">
+      <h2>Become a global citzen!</h2>
+      <h1>Eagle English<br> smart learning <br>for all</h1>
+      <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sequi exercitationem perferendis nemo. Inventore
+        tempore eos perferendis reiciendis vel pariatur?</p>
+      <h3>Be part of our CAST</h3>
+      <span>"insert a quotation blablablabla" - author</span>
+    </section>
+
+    <!-- COURSE FEATURES -->
+    <section id="course-features">
+      <h2>Course Features</h2>
+      <span></span>
+      <ul id="feat-grid">
+        <li>
+          <div> <img src="assets/icons/1.png" alt=""></div>
+          <h3>Custom<br>Paths</h3>
```

Comment:
* [OPTIONAL] 
* I would suggest that you increase the font size of your page content a bit as it currently appears tiny, doing this would enhance your user experience